contrib/buildsystems/engine.pl

```diff
@@ -40,6 +41,7 @@ sub showUsage
 # Parse command-line options
 while (@ARGV) {
     my $arg = shift @ARGV;
+	#print "Arg: $arg \n";
```

Comment:
I think there was some discussion about adding dead code on the mailing list... I would actually agree that it would be better to avoid adding it.